Triangulation_2/include/CGAL/Triangulation_face_base_with_id_2.h

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2019  GeometryFactory (France).  All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org); you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public License as
+// published by the Free Software Foundation; either version 3 of the License,
+// or (at your option) any later version.
+//
+// Licensees holding a valid commercial license may use this file in
+// accordance with the commercial license agreement provided with the software.
+//
+// This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+// WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: LGPL-3.0+
```

Comment:
inconsistent with the package?

Reply:
https://github.com/CGAL/cgal/blob/master/Triangulation_2/include/CGAL/Triangulation_vertex_base_with_id_2.h is LGPL also. Both are meant to be used with the BGL package.